src/icons/Google.tsx

```diff
@@ -3,10 +3,10 @@ import Svg, { Path } from 'react-native-svg'
 import Colors from 'src/styles/colors'
 
 const GoogleIcon = ({ color = Colors.black }) => (
-  <Svg width={17} height={18} viewBox="0 0 17 18" fill="none">
+  <Svg width={32} height={32} fill="none">
     <Path
-      d="M16.7916 8.2513H9.14998V10.5263H14.575C14.3 13.7013 11.6583 15.0596 9.15832 15.0596C5.96665 15.0596 3.16665 12.543 3.16665 9.0013C3.16665 5.58463 5.83332 2.94297 9.16665 2.94297C11.7416 2.94297 13.25 4.58463 13.25 4.58463L14.8333 2.93463C14.8333 2.93463 12.8 0.667969 9.08332 0.667969C4.34998 0.667969 0.69165 4.66797 0.69165 9.0013C0.69165 13.2096 4.13332 17.3346 9.20832 17.3346C13.6666 17.3346 16.9166 14.2763 16.9166 9.75963C16.9166 8.8013 16.7916 8.2513 16.7916 8.2513Z"
       fill={color}
+      d="M23.456 15.28H16.12v2.184h5.208c-.264 3.048-2.8 4.352-5.2 4.352-3.064 0-5.752-2.416-5.752-5.816 0-3.28 2.56-5.816 5.76-5.816 2.472 0 3.92 1.576 3.92 1.576l1.52-1.584S19.624 8 16.056 8C11.512 8 8 11.84 8 16c0 4.04 3.304 8 8.176 8 4.28 0 7.4-2.936 7.4-7.272 0-.92-.12-1.448-.12-1.448Z"
```

Comment:
New icons in designs. To get the spacing looking the same between google and apple it was easiest just to update the google icon to the latest.